llvm/lib/Target/BPF/BPFRegisterInfo.cpp

```diff
@@ -52,22 +52,11 @@ static void WarnSize(int Offset, MachineFunction &MF, DebugLoc& DL)
   OldMF = &(MF.getFunction());
   int MaxOffset = -1 * BPFRegisterInfo::FrameLength;
   if (Offset <= MaxOffset) {
-    if (MF.getSubtarget<BPFSubtarget>().isSolana()) {
-      dbgs() << "Error:";
-      if (DL) {
-        dbgs() << " ";
-        DL.print(dbgs());
-      }
-      dbgs() << " Function " << MF.getFunction().getName() << " Stack offset of " << -Offset
```

Comment:
Don't we want to keep displaying this message for fixed solana stack frames?

Reply:
Yup nice catch, fixed